src/components/Staking/StakingStatsBox.tsx

```diff
@@ -80,47 +92,64 @@ const StatsBoxGrid: React.FC<IProps> = () => {
         } = await getData<{
           data: { totalvalidatorbalance: number; validatorscount: number }
         }>("https://mainnet.beaconcha.in/api/v1/epoch/latest")
-
         const valueTotalEth = formatInteger(
           Number((totalvalidatorbalance * 1e-9).toFixed(0))
         )
         const valueTotalValidators = formatInteger(validatorscount)
-        const currentAprDecimal = calculateStakingRewards(
-          totalvalidatorbalance * 1e-9
-        )
-        const valueCurrentApr = formatPercentage(currentAprDecimal)
         setTotalEth(valueTotalEth)
         setTotalValidators(valueTotalValidators)
-        setCurrentApr(`~${valueCurrentApr}`)
-        setError(false)
       } catch (error) {
-        setTotalEth("n/a")
-        setTotalValidators("n/a")
-        setCurrentApr("n/a")
-        setError(true)
+        setTotalEth(null)
+        setTotalValidators(null)
+      }
+    })()
+    ;(async () => {
+      try {
+        const {
+          data: { apr },
+        } = await getData<{
+          data: { apr: number }
+        }>("https://beaconcha.in/api/v1/ethstore/650")
+        const valueCurrentApr = formatPercentage(apr)
+        setCurrentApr(valueCurrentApr)
+      } catch (error) {
+        setCurrentApr(null)
       }
     })()
   }, [intl.locale])
 
-  // TODO: Improve error handling
-  if (error) return <StatErrorMessage />
-
   return (
     <Container>
       <Cell>
-        <Value>{totalEth}</Value>
+        {totalEth === ZERO ? (
+          <Spinner />
```

Comment:
What color should the spinners here be @nloureiro ([link to page](https://ethereumorgwebsitedev01-aprendpoint.gtsb.io/en/staking))? The same color as the item it's rendering?

<img width="849" alt="Screenshot 2022-11-04 at 09 16 31" src="https://user-images.githubusercontent.com/62268199/199936424-2c65c94d-5b2d-44c0-9448-283bce146976.png">

<img width="654" alt="Screenshot 2022-11-04 at 09 18 06" src="https://user-images.githubusercontent.com/62268199/199936679-c3558a4e-94d3-48d2-9d5e-51ee73dae2de.png">

Reply:
I think we can reduce the spinners a little bit to `--spinner-size: 1rem;`

For the color, I would go to `--eth-colors-gray-200` #E2E8F0

The other thing that loud improves is to give a fixed height to avoid the scroll jump in the pages when the numbers load and the spinners get out. dunno if it's within the scope of this PR

![Screen Shot 2022-11-04 11 39 36 AM](https://user-images.githubusercontent.com/1120748/199964080-a16810a4-e47c-40d7-8f49-c502f044c2b7.png)

Reply:
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/54227730/200066948-64c792b6-3c06-4925-805d-d752282f2ea6.png">

This is what I see on light mode when I use `gray.200` (#E2E8F0)

If we want a dimmed color on both light and dark mode, I would suggest just using the "disabled" semantic color token

Dark mode with first option using "disabled" (other two using "gray.200")
<img width="552" alt="image" src="https://user-images.githubusercontent.com/54227730/200067417-ccb9e310-eb20-491f-8f4c-8242e7f09475.png">
Light mode, same setup:
<img width="517" alt="image" src="https://user-images.githubusercontent.com/54227730/200067482-dd84c062-f71e-4736-81bb-ec6f58788064.png">

Reply:
I would address a minHeight separately, perhaps as part of the hero revamp that will include a subset for this type of hero

Reply:
ok. deal. I will add this to the design system

Reply:
Created an issue for this in #8516.

Merging this PR as it's a huge UX win over what we currently have.